polytope/polytope.py

```diff
@@ -171,8 +170,8 @@ def __str__(self):
             spacer = ' |    '
 
             if mid_ind > 1:
-                output += '\n  '.join([A_rows[k]+spacer+b_rows[k] \
-                                        for k in xrange(mid_ind)]) + '\n'
+                output += '\n  '.join([A_rows[k]+spacer+b_rows[k]
+                                      for k in xrange(mid_ind)]) + '\n'
```

Comment:
thoughts? @carterbox

I can make the change following the merge if that is easier for you. I already made the change locally during my review.